src/webpack/plugin/siteLoaderUtils.js

```diff
@@ -22,6 +25,21 @@ const getMdxLoader = (publicPath) => ({
         }
         return url.href;
       }],
+      [rehypeUrlInspector, {
+        inspectEach: (url) => {
+          if ((url.url.includes('https') || url.url.includes('http')) && !url.file.history[0].includes('node_modules')) {
+            const xmlHttp = new XMLHttpRequest();
+            xmlHttp.open('GET', url.url, false);
+            xmlHttp.onloadend = () => {
+              if (xmlHttp.status === 404 && new RegExp(/Not Found/gi).test(xmlHttp.responseText)) {
+                console.warn('Warning! Broken Link', url.url, 'in', url.file.history[0], 'at line:', url.node.position.start.line);
```

Comment:
You may be able to access the webpack logger and pass it here to use instead of the console.
https://webpack.js.org/api/logging/

Reply:
Updated here https://github.com/cerner/terra-dev-site/pull/330/commits/cef108bcdeb40393bce44105c8f70ce21d06c956.